src/common/options.h

```diff
@@ -76,6 +76,8 @@ struct Option {
 
   unsigned flags = 0;
 
+  int subsys = -1; // if >= 0, we are a subsys debug level
```

Comment:
Makes me itch to subclass Option (and use pointers everywhere to enable doing that) for special bits like this, but that's for a rainy day in the future.